squape/report.py

```diff
@@ -216,8 +216,9 @@ def fatal(msg: str, details: str = "") -> None:
 def enable_loglevel_in_test_module() -> None:
     """Adds support for log levels to the Squish 'test' module.
 
-    DISCLAIMER: This function uses monkeypathching
-    https://en.wikipedia.org/wiki/Monkey_patch
+    !!! warning
+        This function uses monkey pathching
+        https://en.wikipedia.org/wiki/Monkey_patch
```

Comment:
Will those changes be present in the sources too?

Reply:
I don't understand the question.